src/main/groovy/de/dkfz/roddy/execution/cluster/pbs/PBSCommand.groovy

```diff
@@ -147,6 +147,10 @@ class PBSCommand extends Command {
 
         StringBuilder qsubCall = new StringBuilder(EMPTY)
 
+        if (job.getToolScript()) {
```

Comment:
I also don't know if this will work.
I tested in in a groovy console with:

```
def text="#/bin/bash\necho newline\necho newline2" 

def proc = "echo ${text} | qsub".execute()
proc.waitFor()
println proc.inputStream.readLines()
println proc.exitValue()
```